lib/menu.rb

```diff
@@ -0,0 +1,22 @@
+class Menu 
+
+	attr_reader :menu
+
```

Comment:
To get access the list of dishes you would have to call the method menu.menu, which is kind of awkward. Convention for an instance variable with an array of dishes would be to call the array @dishes (or an array of line items would be called @line_items).